src/components/tutorial/FeedbackButton.astro

```diff
@@ -25,7 +25,12 @@ const instanceId = Math.round(Math.random() * 10e9).toString(36);
 ---
 
 <feedback-form>
-	<button data-open-modal disabled>
+	<button id="open-modal-btn" data-open-modal disabled>
```

Comment:
Sounds good! I think with some of these things I didn't want to change the styles in there in case the element was being used else where. In hindsight, this is a good change since there is only one `open-modal` button.